xonsh/environ.py

```diff
@@ -1261,10 +1261,8 @@ def xonshrc_context(rcfiles=None, execer=None, ctx=None):
     return ctx
 
 
-def windows_env_fixes(ctx):
+def windows_foreign_env_fixes(ctx):
     """Environment fixes for Windows. Operates in-place."""
-    # Windows default prompt doesn't work.
-    ctx['PROMPT'] = DEFAULT_PROMPT
     # remove these bash variables which only cause problems.
     for ev in ['HOME', 'OLDPWD']:
```

Comment:
HOME is still useful - from Windows. The way I've tweaked it, HOME can still be inherited from the Windows environment, but on Windows it won't be pulled in from bash, because apparently that causes problems (which I can well believe).